corehq/apps/auditcare/management/commands/gdpr_scrub_user_auditcare.py

```diff
@@ -18,10 +14,5 @@ def add_arguments(self, parser):
         parser.add_argument('username', help="Username to scrub")
 
     def handle(self, username, **options):
-        def update_username(event_dict):
-            event_dict['user'] = new_username
-            return DocUpdate(doc=event_dict)
-
-        new_username = "Redacted User (GDPR)"
-        event_ids = navigation_event_ids_by_user(username)
-        iter_update(NavigationEventAudit.get_db(), update_username, with_progress_bar(event_ids, len(event_ids)))
+        events = navigation_events_by_user(username)
+        events.query.update(user="Redacted User (GDPR)")
```

Comment:
Not sure if I'm reading this incorrectly but it looks like this is updating only what's in SQL. If we had a GDPR request before we were done migrating auditcare, we'd have to remember to re-add the version that scubs it in couch. From what I know those requests are very infrequent, but we also don't yet have a particular timeline in mind to be done migrating. Might it make sense to keep the old code for couch and just add the new code to it for SQL as well, until the migration's done?

That's kind of related to my general question of what the plan is for accessing data in couch once the models are deleted (below).

Reply:
87ac2df454b4bcf0ebd623dbd208f014eba546b0